HW_1/Task_4.py

```diff
@@ -0,0 +1,16 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. Сколько журавликов сделал 
+# каждый ребенок, если известно, что Петя и Сережа сделали 
+# одинаковое количество журавликов, а Катя сделала в два раза 
+# больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+print('Введите количество журавликов')
+quant = int(input())
```

Comment:
Функцию `print` использовать не обязательно. Сообщения для ввода можно поместить внутрь функции `input`.
```suggestion
quant = int(input('Введите количество журавликов'))
```

Reply:
Как я говорил на семинаре необходимо проверять входные данные на корректность, иначе мы можем получить нерелевантные результаты, которые не будут соотноситься с этими входными данными. Например, сумма журавликов, полученных по результатам может оказаться меньше того количества, которое мы вы, которое было указано на входе.